ReqIFSharp/AttributeValue/AttributeValueDate.cs

```diff
@@ -162,10 +165,14 @@ public override void WriteXml(XmlWriter writer)
         {
             if (this.Definition == null)
             {
-                throw new SerializationException("The Definition property of an AttributeValueDate may not be null");
+                return;
```

Comment:
we are introducing some potentially unexpected behaviour i think.

An API developer forgets to set the `Definition` property. What we do here is that we skip the serialization of the attribute. The resulting document may be correct from a schema point of view, but the content is probably not what the API developer expects. This is the reason we have these `SerializarionExption`s in the WriteXml methods. I would have to investigate if we can leverage schema validation at serialization, or wether this is only supported during deserialization

Reply:
Okay, I change it